Sasquatch/Config/Sasquatch.xcconfig

```diff
@@ -1,5 +1,12 @@
 #include "../../Config/App.xcconfig"
 
+ARCHS[sdk=iphoneos*] = armv7 armv7s arm64 arm64e
+ARCHS[sdk=watchos*] = armv7k arm64_32
```

Comment:
We have `MS_WATCH_ARCHS` defined in `Global.xcconfig`. Let's add `arm64_32` to global config and use it.

Reply:
I've added that.